backend/api/tests/test_models.py

```
@@ -0,0 +1,156 @@
+import pytest
+from django.utils import timezone
+# ValueError is a built-in exception, no need to import from django.core.exceptions
+from api.models import User, Video, CompanyProfile, ViewerProfile
+
+# Mark all tests in this module to use the database
+pytestmark = pytest.mark.django_db
+
+# --- User Model Tests ---
+
+def test_create_user():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_create_user`

Add return type annotation: `None`

(ANN201)